src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -371,7 +371,10 @@ public async Task SslStream_ClientCertificate_SendsChain()
                 chain.ChainPolicy.DisableCertificateDownloads = false;
                 bool chainStatus = chain.Build(clientCertificate);
                 // Verify we can construct full chain
-                Assert.True(chain.ChainElements.Count >= clientChain.Count, "chain cannot be built");
+                if (chain.ChainElements.Count < clientChain.Count)
```

Comment:
You said "for now" in the description. Is there an issue this should be tracked against?

Reply:
This will leave #48091 open so that should be used for tracking.